zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/client/DocumentenApiClient.kt

```diff
@@ -72,6 +77,78 @@ class DocumentenApiClient(
         return result
     }
 
+     fun storeDocumentInParts(
+        authentication: DocumentenApiAuthentication,
+        baseUrl: URI,
+        request: BestandsdelenRequest,
+        createDocumentResult: CreateDocumentResult,
+        bestandsnaam: String?
+    ): BestandsdelenResult {
+         requireNotNull(bestandsnaam) { "Bestandsnaam must be set otherwise uploading in bestanddelen will fail" }
+
+         // Inside the CreateDocumentResult there is an array of bestandsdelen.
+         // Each bestandsdeel needs to be sent separately
+         // So the documenten api determines the amount of chunks, not this application.
+         var response: BestandsdelenResult? = null
+
+         logger.info("Starting upload of file in {} chunks\n", createDocumentResult.bestandsdelen?.size)
```

Comment:
```suggestion
         logger.info { "Starting upload of file $bestandsnaam in ${createDocumentResult.bestandsdelen?.size} chunks" }
```

Reply:
I will change the log and I will check the other calls. Is info the correct level or is debug better?

Reply:
It's hard to say. In general we want to log changes to external systems as info, but we also don't want an overload of information. If anything this one is probably fine, and the log line per chunk might have to be debug. @marijnritense @Klaas-Ritense what do you think?

Reply:
I made the first log line info and the line for each chunk (and size) debug